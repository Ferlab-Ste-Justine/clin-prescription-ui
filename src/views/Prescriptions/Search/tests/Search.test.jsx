import React from 'react';
import renderer from 'react-test-renderer';

import PatientSearchWrapper from '../index';

describe('PatientSearchWrapper', () => {
  test('should be as the snapshot', () => {
    const component = renderer.create(<PatientSearchWrapper />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
