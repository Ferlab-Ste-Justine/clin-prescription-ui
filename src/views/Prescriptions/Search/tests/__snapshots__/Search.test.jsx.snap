// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PatientSearchWrapper should be as the snapshot 1`] = `null`;
